### ISSUE TITLE
Building with rav1e throws error

### ISSUE THREAD

**Issue by adityapatadia** [strukturag/libheif#290]
I am trying to build with `rav1e` support with `./configure --enable-local-rav1e`. However, I am getting following error.

```
heif_encoder_rav1e.cc:40:10: fatal error: rav1e/rav1e.h: No such file or directory
3565 #include "rav1e/rav1e.h"
```

Can you let me know if there is any issue?

**Comment by adityapatadia** [comment]
It would be great if this library builds with local rav1e after just installing it with `cargo install`.

**Comment by farindk** [comment]
The error you got (your first comment) is probably because you first have to compile rav1e in the `third-party` subdirectory. There is a script `rav1e.cmd` there, which should help with this.

I hope that there will soon be official packages for rav1e that we could depend on.

**Comment by adityapatadia** [comment]
I used the same script and still get this error. I compile like this.
```sh
chmod +x ./third-party/rav1e.cmd && ./third-party/rav1e.cmd && \
  ./autogen.sh && ./configure --enable-local-rav1e && \
  make -j2 V=0 && make install
```

rav1e official packages are available as of version 0.3.3: https://crates.io/crates/rav1e
